unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/AudioStream/ECSAudioStreamComponentHandler.cs

```diff
@@ -23,17 +23,14 @@ public class ECSAudioStreamComponentHandler : IECSComponentHandler<PBAudioStream
         // Flags to check if we can activate the AudioStream
         internal bool isInsideScene = false;
         internal bool isRendererActive = false;
-        internal bool wasCursorLocked = false;
+        internal bool hadUserInteraction = false;
```

Comment:
`hadUserInteraction` could mean a lot of stuffs, but I'm not sure what could be a proper name for that field 🤷‍♂️

Reply:
haha yeah but for me that's the most clear name for what's needed for the freaking video to start playing in the browser... user interaction.